src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js

```diff
@@ -74,6 +74,7 @@ class CJ4_FMC extends FMCMainDisplay {
         this.maxCruiseFL = 450;
         this.onFplan = () => { CJ4_FMC_RoutePage.ShowPage1(this); };
         this.onLegs = () => { CJ4_FMC_LegsPage.ShowPage1(this); };
+        this.onMsg = () => { CJ4_FMC_MsgPage.ShowPage11(this); };
```

Comment:
This looks like it calls a `ShowPage 11()` method instead of `ShowPage1()`.

Reply:
Thank you! I have bad butter fingers :) Will fix it right away! Also would love to know if there's a clear TO-DO list other than just going through the issues?

Reply:
> Thank you! I have bad butter fingers :) Will fix it right away! Also would love to know if there's a clear TO-DO list other than just going through the issues?

We have a Trello:
https://trello.com/b/3GytX6nN/cj4

Feel free to choose something :)